src/ActionBar/ActionBarActions.js

```diff
@@ -0,0 +1,19 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ActionBarActions = ({ children, className, ...props }) => {
+    const actionBarActionsClasses = classnames(
+        'fd-action-bar__actions',
+        className
+    );
+
+    return <div {...props} className={actionBarActionsClasses}>{children}</div>;
+};
+
+ActionBarActions.propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string
+};
+
+export default ActionBarActions;
```

Comment:
When it makes sense, I thought we talked about naming the subcomponent files with a leading underscore (_e.g._ `_ActionBarActions`), not exporting them as top-level standalone components, but then adding static references to them in the main component (_e.g._ `ActionBar.Actions = ActionBarActions`).  This way, the consumer just has to import `ActionBar` and they automatically get it and all the subcomponents together.

I don't think this applies unconditionally across the board as we probably wouldn't want to have to code tabs as `TabGroup.Tab` when `Tab` could just be exported.

That said, I think this component is the perfect candidate for making the subcomponents private static references.

Reply:
Forgot about that approach. That cleaned things up quite a bit as well. Fixed in 7f9264d